.gitattributes

```diff
@@ -0,0 +1,2 @@
+poetry.lock binary linguist-generated
```

Comment:
Why are we ignoring `poetry.lock`? We could, of course, just remove it completely from the repo and continue without it, but this is against best practice (https://python-poetry.org/docs/basic-usage/#commit-your-poetrylock-file-to-version-control).